crates/nu-cli/src/completions/variable_completions.rs

```diff
@@ -9,6 +9,10 @@ use nu_protocol::{
 use reedline::Suggestion;
 use std::str;
 
+use super::{completion_common::sort_suggestions, SortBy};
+
+const SORT_BY: SortBy = SortBy::Ascending;
```

Comment:
I'd prefer to have SortBy::Ascending below instead of SORT_BY. I don't think this provides much value. What am I missing?

Reply:
It's there in case we decide to change the sort order in the future, so we only have to change this one constant rather than update all four calls to sort_suggestions. It's unlikely that someone would forget, though, so I can go either way on this

Reply:
ya, i figured it was for something like that. I just think it adds extra bloat. others may differ. either way, it's a good pr. thanks.